src/Generators/Microsoft.Gen.Logging/Parsing/Parser.Records.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.Gen.Shared;
+
+namespace Microsoft.Gen.Logging.Parsing
+{
+    internal partial class Parser
```

Comment:
Is this internal for testing? Otherwise it can be private.

Reply:
`Parser` class was `internal` even before my changes